contractcourt/channel_arbitrator.go

```diff
@@ -851,11 +899,14 @@ func (c *ChannelArbitrator) stateStep(triggerHeight uint32,
 
 		// With the commitment broadcast, we'll then send over all
 		// messages we can send immediately.
-		err = c.cfg.DeliverResolutionMsg(pktsToSend...)
-		if err != nil {
-			// TODO(roasbeef): make sure packet sends are idempotent
-			log.Errorf("unable to send pkts: %v", err)
-			return StateError, closeTx, err
+		if len(pktsToSend) != 0 {
```

Comment:
`DeliverResolutionMsg` is already a NOP for zero-length slices. if this is only for satisfying the tests, perhaps we update the tests to ignore empty slices instead of vice versa?

Reply:
Yeah this was added just so the test could pass. Once could argue that that current check is an underlying detail of the default `DeliverResolutionMsg` closure, so it's simply a defensive check on the part of the channel arb.

Reply:
On the other hand, one could argue that we lose a valuable testing hook, that could otherwise be used to assert the internal state machine is progressing as expected ;)